.github/workflows/ci.yaml

```diff
@@ -75,13 +75,13 @@ jobs:
         version: ${{ env.kind-version }}
         image: ${{ matrix.kind-image }}
         wait: 300s
-    - name: Wait for cluster to finish bootstraping
-      run: kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout=300s
```

Comment:
We need this block to ensure cluster is up and running before adding more manifests.

Reply:
On empty cluster? what pods?

Reply:
Ok.